agent/src/agent/pipeline/pipeline.py

```diff
@@ -68,8 +68,9 @@ class Pipeline(Entity):
     name = Column(String)
     source_id = Column(Integer, ForeignKey('sources.id'))
     destination_id = Column(Integer, ForeignKey('destinations.id'))
-    config = Column(MutableDict.as_mutable(JSON))
-    override_source = Column(MutableDict.as_mutable(JSON))
+    config = Column(JSON)
+    schema = Column(JSON)
+    override_source = Column(JSON)
```

Comment:
The type of these fields must be MutableDict.as_mutable(JSON), otherwise, you won't be able to update their values

Reply:
We don't need to change the object itself. We can set a new value if we need to change something. Anyway, it's better not to change it at all.
Without these MutableDict there are no strange updates anymore so it fixes our deadlock problem.

Reply:
then do we need to update other entities?